src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -536,21 +536,21 @@ protected function buildCommand($binary, $input, $output, array $options = [])
                 } elseif (\is_array($option)) {
                     if ($this->isAssociativeArray($option)) {
                         foreach ($option as $k => $v) {
-                            $command .= ' --' . $key . ' ' . \escapeshellarg($k) . ' ' . \escapeshellarg($v);
+                            $command .= ' --' . $key . ' ' . \escapeshellarg($k) . ' ' . \escapeshellarg($v ?? '');
                         }
                     } else {
                         foreach ($option as $v) {
-                            $command .= ' --' . $key . ' ' . \escapeshellarg($v);
+                            $command .= ' --' . $key . ' ' . \escapeshellarg($v ?? '');
```

Comment:
I need to double check on the wkhtmltopdf options but, IMHO, we should simply skip invalid array entries.
I think we should skip options with key/value that are not valid strings.

Reply:
I'm saying to skip it as it's the current behavior for null/false options.